tools/build-jetpack.sh

```diff
@@ -72,7 +72,7 @@ hash yarn 2>/dev/null || {
     exit 1;
 }
 yarn --cwd $TARGET_DIR cache clean
-yarn --cwd $TARGET_DIR run build
+COMPOSER_MIRROR_PATH_REPOS=1 yarn --cwd $TARGET_DIR run build
```

Comment:
👍 This seems to work well for me:

>   - Installing automattic/jetpack-logo (dev-update/dont-symlink-packages): Mirroring from ./packages/logo

while without it for local dev I properly get:

>   - Installing automattic/jetpack-logo (dev-update/dont-symlink-packages): Symlinking from ./packages/logo